### ISSUE TITLE
How to fix this error? Plz help

### ISSUE THREAD

**Issue by Affanabbbas** [vita-epfl/detection-attributes-fields#5]
usage: python3 -m openpifpaf.train [options]
python3 -m openpifpaf.train: error: **unrecognized arguments:** --jaad-root-dir C:/Users/VR/Desktop/JAAD_clips.zip --jaad-subset default --jaad-training-set train --jaad-validation-set val --pifpaf-pretraining --detection-bias-prior 0.01 --jaad-head-upsample 2 --jaad-pedestrian-attributes all --fork-normalization-operation power --fork-normalization-duplicates 35 --attribute-regression-loss l1 --attribute-focal-gamma 2

I just gave the path to the output of the model and the data set this way.
Please take a look:

!python3 -m openpifpaf.train \
  --output C:/Users/VR/Desktop/model \
  --dataset jaad \
  --jaad-root-dir C:/Users/VR/Desktop/JAAD_clips.zip \
  --jaad-subset default \
  --jaad-training-set train \
  --jaad-validation-set val \
  --log-interval 10 \
  --val-interval 1 \
  --epochs 5 \
  --batch-size 4 \
  --lr 0.0005 \
  --lr-warm-up-start-epoch -1 \
  --weight-decay 5e-4 \
  --momentum 0.95 \
  --basenet fn-resnet50 \
  --pifpaf-pretraining \
  --detection-bias-prior 0.01 \
  --jaad-head-upsample 2 \
  --jaad-pedestrian-attributes all \
  --fork-normalization-operation power \
  --fork-normalization-duplicates 35 \
  --lambdas 7.0 7.0 7.0 7.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 1.0 \
  --attribute-regression-loss l1 \
  --attribute-focal-gamma 2 \
  --auto-tune-mtl

**Comment by Affanabbbas** [comment]
I have tried both, the zip file and the extracted one
